spring-ai-core/src/main/java/org/springframework/ai/chat/observation/ChatModelCompletionObservationFilter.java

```diff
@@ -36,7 +37,7 @@ public Observation.Context map(Observation.Context context) {
 
 		chatModelObservationContext
 			.addHighCardinalityKeyValue(ChatModelObservationDocumentation.HighCardinalityKeyNames.COMPLETION
-				.withValue(ChatModelObservationContentProcessor.concatenateStrings(completions)));
+				.withValue(TracingHelper.concatenateStrings(completions)));
```

Comment:
Extracted the logic into the `TracingHelper` to re-use it for the vector store observations.